mail/urlscan/Portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        firecat53 urlscan 0.9.5
+categories          mail python
+platforms           darwin
+license             GPL-2
+maintainers         {@ryanakca debian.org:rak}
+description         extract and browse the URLs in an email
+long_description    ${name} lets you easily browse URLs contained in an email\
+                    and launch a web browser to view them. It is a replacement\
+                    for the "urlview" program and integrates with the mutt MUA.
+
+github.tarball_from archive
+
+python.default_version     38
```

Comment:
does this work with Python 3.9; if so please use that version (this is now used as the default version in the python PG)

Reply:
Unfortunately not yet. The dependency py39-urwid is not yet a port AFAICT.